src/routes/work/continuous-delivery-workshop/_waste.svelte

```diff
@@ -62,12 +62,12 @@
 
 <dl class="grid grid-cols-1 gap-y-2 md:grid-cols-2 md:gap-y-6 md:gap-x-8">
 	{#each wastes as waste, index}
-		<div class="flex flex-col gap-y-2">
+		<div class="flex flex-col m-2 p-1">
 			<button on:click={() => (showWastes[index] = !showWastes[index])}>
 				<dt class="flex items-center gap-x-4">
 					<img class="object-contain w-12 h-12" src={waste.img} alt="waste-type" />
 					<div class="group flex items-center gap-x-4">
-						<h4 class="mb-0 group-hover:text-v-lilac">{waste.name}</h4>
+						<p class="mb-0 group-hover:text-v-lilac text-xl font-medium text-black">{waste.name}</p>
```

Comment:
We'd like to keep using the `h1-4` elements to ensure consistency and avoid copying the `text-xl font-medium text-black` classes across the whole site...

The cause of all this was switching to using the tailwind [typography plugin](https://tailwindcss.com/docs/typography-plugin) across the whole site...

Reply:
I had some queries / clarifications ( if you don't mind ) ...

- Heading elements  h1-h6 [should not be used to resize text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes) hence I used `p` instead of `h4`.
- To use consistent text color, a [grayscale can be chosen](https://tailwindcss.com/docs/typography-plugin#choosing-a-gray-scale) for `prose`, and copying text-black won't be required.
- If same style is to be used at several places, Shouldn't a component be created ?

Nevertheless, I have adjusted the padding to fix misalignments.